ssr/src/page/profile/mod.rs

```diff
@@ -183,7 +191,7 @@ pub fn ProfileView() -> impl IntoView {
                 user_details
                     .get()
                     .map(|user_details| {
-                        view! { <ProfileComponent user_details/> }
+                        view! { <ProfileComponent is_native_profile=false user_details /> }
```

Comment:
It will not always be false you might also land on your profile if  the path `/profile/<principal-id>`  matches your principal.

Reply:
Done